index.html

```diff
@@ -0,0 +1,88 @@
+                                fill="white" />
+                            <path
+                                d="M976.194 138.605V189H948.062V100.636H974.813V116.859H975.791C977.747 111.452 981.084 107.214 985.801 104.146C990.518 101.039 996.137 99.4858 1002.66 99.4858C1008.87 99.4858 1014.26 100.886 1018.82 103.685C1023.42 106.447 1026.99 110.32 1029.52 115.306C1032.09 120.254 1033.36 126.045 1033.32 132.68V189H1005.19V138.202C1005.23 133.293 1003.98 129.458 1001.45 126.697C998.956 123.935 995.485 122.555 991.036 122.555C988.083 122.555 985.475 123.207 983.212 124.511C980.988 125.776 979.262 127.598 978.035 129.976C976.846 132.354 976.232 135.23 976.194 138.605Z"
+                                fill="white" />
+                            <path
+                                d="M1083.94 190.668C1074.7 190.668 1066.72 188.847 1060.01 185.203C1053.34 181.521 1048.2 176.286 1044.59 169.498C1041.03 162.671 1039.24 154.56 1039.24 145.163C1039.24 136.036 1041.04 128.058 1044.65 121.232C1048.25 114.366 1053.34 109.036 1059.89 105.239C1066.45 101.403 1074.18 99.4858 1083.08 99.4858C1089.37 99.4858 1095.12 100.464 1100.34 102.42C1105.55 104.376 1110.06 107.271 1113.86 111.107C1117.65 114.942 1120.61 119.678 1122.72 125.316C1124.82 130.915 1125.88 137.339 1125.88 144.588V151.607H1049.08V135.268H1099.7C1099.67 132.277 1098.96 129.612 1097.58 127.272C1096.2 124.933 1094.3 123.111 1091.88 121.807C1089.5 120.464 1086.76 119.793 1083.65 119.793C1080.51 119.793 1077.69 120.503 1075.2 121.922C1072.7 123.303 1070.73 125.201 1069.27 127.617C1067.81 129.995 1067.05 132.699 1066.97 135.729V152.354C1066.97 155.96 1067.68 159.124 1069.1 161.847C1070.52 164.531 1072.53 166.621 1075.14 168.117C1077.75 169.613 1080.85 170.361 1084.46 170.361C1086.95 170.361 1089.21 170.016 1091.25 169.325C1093.28 168.635 1095.03 167.619 1096.48 166.276C1097.94 164.934 1099.03 163.285 1099.76 161.329L1125.59 162.077C1124.52 167.868 1122.16 172.911 1118.52 177.207C1114.91 181.464 1110.17 184.781 1104.31 187.159C1098.44 189.499 1091.65 190.668 1083.94 190.668Z"
+                                fill="white" />
+                        </svg></a>
+                    </div>
+                    <!-- Username input -->
+                    <div class="col-12 mb-1">
+                        <label class="visually-hidden" for="inlineFormInputGroupUsername">Username</label>
+                        <div class="input-group">
+                            <div class="input-group-text">@</div>
+                            <input type="text" class="form-control" id="inlineFormInputGroupUsername"
```

Comment:
```suggestion
                            <input required type="text" class="form-control" id="inlineFormInputGroupUsername"
```

HTML Form validation is required